src/main/java/com/flexrate/flexrate_back/member/dto/PatchMemberResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.flexrate.flexrate_back.member.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.flexrate.flexrate_back.member.enums.MemberStatus;
+import com.flexrate.flexrate_back.member.enums.Sex;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Getter
+@AllArgsConstructor
+@Builder
+public class PatchMemberResponse {
+    private Long memberId;
+    private String name;
+    private Sex sex;
+    private LocalDate birthDate;
+
+    private MemberStatus memberStatus;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
+    private LocalDateTime updatedAt;
+}
```

Comment:
기존 코드들을 참고하시면, DTO들에는 record를 사용하고있어요!
record는 불변 객체로, 생성 후 내부 상태를 변경할 수 없어 데이터의 일관성을 보장해주고, 다중 스레드 환경에서 안전하게 사용할 수 있도록 해줘용.
또한 getter도 자동으로 생성해주니 @Getter를 선언하지 않아도 돼요.
컨벤션을 맞출수있도록 변경해주세요!

```suggestion
@Builder
public record PatchMemberResponse (
        Long memberId,
        String name,
        Sex sex,
        LocalDate birthDate,
        MemberStatus memberStatus,

        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
        LocalDateTime updatedAt
) {}
```